packages/plugin/src/theia.d.ts

```diff
@@ -4613,6 +4651,7 @@ export module '@theia/plugin' {
          * Additional information the webview view being resolved.
          *
          * @param T Type of the webview's state.
+         * @deprecated since 1.27.0 - Use [WebviewViewResolveContext](#WebviewViewResolveContext) (from root namespace) instead.
```

Comment:
I believe we can remove this entry completely, it actually fails linting during CI since we define it twice.
There is no need to keep the deprecation afaik?

Reply:
I will gladly remove it. I just thought, that some Theia plugins might use the type somewhere. But i guess this is dependent on the Theia version to plugins specifies so it should not be an issue.